src/query/expression/src/schema.rs

```diff
@@ -237,7 +245,12 @@ impl DataSchema {
     }
 
     pub fn to_arrow(&self) -> ArrowSchema {
-        let fields = self.fields().iter().map(|f| f.into()).collect::<Vec<_>>();
+        let fields = self
```

Comment:
So some fields may be missed when converting to arrow.

The schema convert from arrow may be incomplete.

Reply:
The columns stored in parquet do not include virtual computed columns. A virtual computed column is also not required when moving from a parquet back to a data block. Virtual computed columns are converted to an expression when binder like other expressions.